CHANGELOG.md

```diff
@@ -1,23 +1,32 @@
 # Unreleased
- 
+
 ## New features
 
-### Replace back link placeholder URLs with javascript
+### Replace back link placeholder URLs with JavaScript
 
-Previously, users had to manually replace the `href` for each back link in the page templates.
+You can now use JavaScript to make the [back link component]( https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default.
 
-Now, we use JavaScript to make the back link take you to the previous page by default.
+Before, you had to manually replace the `href` attribute for each back link in the page templates.
 
-This approach is not appropriate for production scenarios, where you cannot rely on JavaScript being available at all times.
+Do not use this approach for production scenarios, where you cannot rely on JavaScript always being available.
 
 You can still override the `href` attribute if you need different behaviour.
```

Comment:
@lfdebrux @vanitabarrett Should we give a 'for example'? Or would that be overkill/unnecessary for some reason?

Reply:
Hm, not sure about this. Our guidance in the design system says "Make sure the link takes users to the previous page they were on, in the state they last saw it." so I don't think we should encourage people to change the behaviour of what the back link does. They may need to override the `href` to provide a solution which works when JavaScript is disabled though/

Reply:
Ah interesting. Should we change to something like this?

```suggestion
You can still override the `href` attribute if you need to provide a solution that works when JavaScript is disabled.
```

Reply:
Sounds good 👍